### PR TITLE
Add a rubocop channel to the codeclimate config

### PR DESCRIPTION
**Pull request by djberg96**
Add a rubocop channel to the codeclimate config

### CODE REVIEW

**Comment by djberg96** [comment]
You can ignore the failure, this build is actually green: https://codeclimate.com/github/ManageIQ/manageiq-smartstate/builds/720

**Comment by djberg96** [comment]
@miq-bot add_reviewer @roliveri

File: .codeclimate.yml [hunk 1]
```diff
@@ -27,29 +25,28 @@ engines:
...
-    # let's enable later
     enabled: false
   markdownlint:
-    # let's enable later
     enabled: false
   rubocop:
     enabled: true
-    config: '.rubocop_cc.yml'
+    config: ".rubocop_cc.yml"
+    channel: rubocop-0-69
```
Comment (roliveri) [inline .codeclimate.yml]: @djberg96 Just making sure... Is this the only non-cleanup change to the file?

Comment (djberg96) [reply .codeclimate.yml]: Yeah, the rest is just the format that `YAML.dump` gives us.

Comment (roliveri) [reply .codeclimate.yml]: OK, good. Are the codeclimate failures the expected failures we discussed yesterday? If so, I'll merge it and cut a new version of the GEM.

Comment (djberg96) [reply .codeclimate.yml]: Yes. Not sure why you need to cut a new version of the gem, though, since it only affects our repo.

**Comment by roliveri** [comment]
That's true. OK, no new GEM version needed.

**Pull request merged**